samples/experimental/stack.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: experimental.kpack.pivotal.io/v1alpha1
+kind: Stack
+metadata:
+  name: cflinuxfs3-stack
+spec:
+  id: "io.buildpacks.stacks.cflinuxfs3"
```

Comment:
Up until this point all of our examples and tutorials have pointed to bionic. Could we keep that?

Reply:
Sure!